tests/PHPStan/Rules/Exceptions/CaughtExceptionExistenceRuleTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Exceptions;
+
+class CaughtExceptionExistenceRuleTest extends \PHPStan\Rules\AbstractRuleTest
```

Comment:
Usually when you have a tool like this, it is better to keep the old class and deprecate it with `@deprecate ..add description here..` in docblock. This way, if somebody extended this class or used it directly, it will not break it's implementation.

And why using FQCN instead of `use` statement for extends and returned type ?